src/Analyzers/CSharp/Analyzers/UseCollectionExpression/CSharpUseCollectionExpressionForArrayDiagnosticAnalyzer.cs

```diff
@@ -131,7 +131,7 @@ private void AnalyzeArrayInitializerExpression(SyntaxNodeAnalysisContext context
         // Have to actually examine what would happen when we do the replacement, as the replaced value may interact
         // with inference based on the values within.
         var replacementCollectionExpression = CollectionExpression(
-            SeparatedList<CollectionElementSyntax>(initializer.Expressions.Select(ExpressionElement)));
+            [.. initializer.Expressions.Select(ExpressionElement)]);
```

Comment:
I didn't expect to see a space between `..` and `initializer` here. Is that included intentionally by the formatter?

Reply:
Yes. Very intentional. Matches what we do with list patterns. Also how we doc this everywhere.  It's not an operator that binds to the thing on the right.  It affects the entire expression, so it needs a space.